_includes/feedback.html

```diff
@@ -1,9 +1,15 @@
 <div class="div-feedback">
     <div class="feedback-header">WAS THIS PAGE HELPFUL?</div>
-    <div>
-        <button id="yes" type="button" value="yes" class="feedback-button">&#10004; Yes</button>
-        <button id="no" type="button" value="no" class="feedback-button">&#10006; No</button>
-        <p class="text-small text-grey-dk-100 hidden">Thank you for your feedback!</p>
+    <div class="feedback-radio-div">
+        <input id="yes" type="radio" name="radio-button" value="yes" class="feedback-button">
+        <label class="feedback-radio" for="yes">&#10004; Yes</label> 
+        <input id="no" type="radio" name="radio-button" value="no" class="feedback-button">
+        <label class="feedback-radio" for="no">&#10006; No</label>
     </div>
+    <div class="feedback-text-header">Tell us why</div>
+    <textarea id="comment" class="feedback-text" placeholder="Enter comment" maxlength="350"></textarea>
+    <div class="text-small num-chars" id="num-chars">350 characters left</div>
+    <button id="send" class="send-button" disabled>Send</button>
+    <p class="text-small text-grey-dk-100 hidden" id="thank-you">Thank you for your feedback!</p>
     <p class="text-small text-grey-dk-100">Want to contribute? <a class="feedback-edit" target="_blank" href="https://github.com/opensearch-project/documentation-website/edit/main/{{ page.path }}">Edit this page</a> or <a class="feedback-issue" target="_blank" href="https://github.com/opensearch-project/documentation-website/issues/new?assignees=&labels=untriaged&template=issue_template.md&title=%5BDOC%5D">create an issue</a>.</p>
```

Comment:
'Edit this page' brings me to a 404 page. Should this link work on staging? 'Create an issue' works fine.

Reply:
@hdhalter Are you doing this on your local system? And what page are you doing it from?